TestHelpers.Tests/MockFileTests.cs

```diff
@@ -558,31 +558,6 @@ public void Serializable_can_deserialize()
 #endif
 
 #if NET40
-        [Test]
-        public void MockFile_Encrypt_ShouldEncryptTheFile()
-        {
-            // Arrange
-            const string Content = "Demo text content";
-            var fileData = new MockFileData(Content);
-            var filePath = XFS.Path(@"c:\a.txt");
-            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
-            {
-                {filePath, fileData }
-            });
-
-            // Act
-            fileSystem.File.Encrypt(filePath);
-
-            string newcontents;
-            using (var newfile = fileSystem.File.OpenText(filePath))
-            {
-                newcontents = newfile.ReadToEnd();
-            }
-
-            // Assert
-            Assert.AreNotEqual(Content, newcontents);
-        }
-
         [Test]
         public void MockFile_Encrypt_ShouldSetEncryptedAttribute()
```

Comment:
It probably makes sense to remove all Encrypt/Decrypt tests from MockFileTests.cs -- they seem to be testing the exact same logic, with pretty much the exact same test code. What we're ultimately testing lives in MockFileInfo anyway.

Reply:
The tests do seem to be redundant at first, but on a second thought, they kinda do make sense. If you want to have a blackbox test for `MockFile.Encrypt`, then asserting that the `Encrypted` attribute is set, would be correct. Right now, I don't see another option to assert that those methods work as document.

Reply:
Yeah I can see that. While they both technically call down to the same implementation, I spose it's possible to break something in MockFile before it even gets to MockFileInfo